Podcast/SeriesDetailViewController.swift

```diff
@@ -212,22 +223,24 @@ class SeriesDetailViewController: ViewController, SeriesDetailHeaderViewDelegate
     func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
         let headerView = UIView(frame: CGRect(x: 0, y: 0, width: self.view.frame.width, height: sectionHeaderHeight))
         headerView.backgroundColor = .paleGrey
-        let sectionTitle = UILabel()
-        sectionTitle.text = "All Episodes"
-        sectionTitle.textColor = .charcoalGrey
-        sectionTitle.font = ._14SemiboldFont()
-        sectionTitle.sizeToFit()
-        sectionTitle.frame = CGRect(x: padding, y: sectionTitleY, width: sectionTitle.frame.width, height: sectionTitleHeight)
-        
-        headerView.addSubview(sectionTitle)
-        
-        let separatorUpper = UIView(frame: CGRect(x: 0, y: 0, width: headerView.frame.width, height: 1))
-        let separatorLower = UIView(frame: CGRect(x: 0, y: headerView.frame.height - separatorHeight, width: headerView.frame.width, height: 1))
-        separatorUpper.backgroundColor = .paleGrey
-        separatorLower.backgroundColor = .paleGrey
-        
-        headerView.addSubview(separatorUpper)
-        headerView.addSubview(separatorLower)
+
```

Comment:
why is this all commented out?

Reply:
my bad I thought I deleted it but I forgot...